scaleway/data_source_ipam_ip.go

```diff
@@ -64,8 +70,18 @@ func dataSourceScalewayIPAMIP() *schema.Resource {
 					}
 				},
 			},
-			"region": regionSchema(),
-
+			"tags": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Description: "The tags associated with the IP",
+			},
+			"zonal":           zoneSchema(),
```

Comment:
Should it be zone?

Reply:
referred as `zonal` on [api side](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/ipam/v1/ipam_sdk.go#L397)